website/thaliawebsite/templates/base.html

```diff
@@ -40,6 +40,13 @@
             <link href="{% static "css/main.scss" %}" rel="stylesheet" type="text/x-scss">
         {% endcompress %}
     {% endblock %}
+    {% if APRIL_FOOLS %}
+    <style>
+        body {
+            transform: rotate(.5deg);
```

Comment:
May I suggest

```
import random
from django import template

register = template.Library()

@register.simple_tag
def random_float(a, b=None):
    if b is None:
        a, b = 0, a
    return round(random.uniform(a, b), 2)
```

with 

`{% load random_float %}`

and 

`rotate({% random_float 2 -2 %}deg);`

Reply:
no, because this will be removed in the release after the next one anyway

Reply:
oops sorry, didn't see what you were actually suggesting

Reply:
i think it's fine the way it is though, very subtle